source/NVDAObjects/IAccessible/mscandui.py

```diff
@@ -213,10 +213,48 @@ def event_show(self):
 		item=MSCandUIWindow_candidateListItem(IAccessibleObject=self.IAccessibleObject,IAccessibleChildID=3)
 		reportSelectedCandidate(item)
 
+class ModernCandidateUICandidateItem(BaseCandidateItem):
+
+	def _get_parent(self):
+		# Candidate list in Microsoft Quick cannot be obtained in IAccessible _get_parent. Use _get_parent in NVDAObject.window
+		parent=super(IAccessible,self).parent

```

Comment:
Mmm. You're saying that you want to call _get_parent on Window, but you are calling super. I know that it is effectively the same, but I'd rather have you call Window instead if you want to make sure that this is not ambiguous. Note that you have to add an import.

```suggestion
		parent=NVDAObjects.window.Window._get_parent(self)
```